### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by pabsilva** [luho383/csg-unity#2]
Hi there!

What is the license for this code? You didn't include a LICENSE.txt (or LICENSE.md)...

**Comment by suprafun** [comment]
Sorry to bother you. Just wanted to bump this.

**Comment by andybak** [comment]
Yeah. I was wondering the same thing. Sadly I can't touch code with a 10 foot pole unless I know the licence.
